charms/kserve-controller/tests/integration/mlserver-sklearn-iris.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "mlserver-sklearn-iris"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model"
+      runtime: kserve-mlserver
+      resources:
+        limits:
+          cpu: 250m
```

Comment:
Maybe you want to increase this a bit more, maybe 1 as 250m is too little for an inference service

Reply:
`cpu: 1` causes test to fail in GH runner. Reducing it to 250m ensures there are enough resources to deploy model.
Log messages when running with `cpu: 1` in GH runner:
```
 test_charm:test_charm.py:308 mlserver-sklearn-iris is not ready {'lastTransitionTime': '2023-09-14T19:56:19Z', 'message': 'Revision "mlserver-sklearn-iris-predictor-default-00001" failed with message: 0/1 nodes are available: 1 Insufficient cpu. preemption: 0/1 nodes are available: 1 No preemption victims found for incoming pod..', 'reason': 'RevisionFailed', 'severity': 'Info', 'status': 'False', 'type': 'PredictorConfigurationReady'}
```

Reply:
I believe that the limit set to 1 and the request set to 250m should work. Have you tried it?